utils/config/tokenrefresh.go

```diff
@@ -55,12 +55,12 @@ func tokenRefreshHandler(currentAccessToken string) (newAccessToken string, err
 	}
 
 	serverConfiguration, err := GetArtifactorySpecificConfig(tokenRefreshServerId, true, false)
```

Comment:
Let's add a description above the definition of the ```tokenRefreshServerId``` variable (at the top of the file).

Reply:
Done